core/src/main/java/org/infinispan/remoting/responses/BiasRevocationResponse.java

```diff
@@ -1,50 +1,30 @@
 package org.infinispan.remoting.responses;
 
-import java.io.IOException;
-import java.io.ObjectInput;
-import java.io.ObjectOutput;
-import java.util.Set;
-
-import org.infinispan.commons.marshall.AdvancedExternalizer;
-import org.infinispan.commons.marshall.Ids;
-import org.infinispan.commons.marshall.MarshallUtil;
-import org.infinispan.commons.util.Util;
+import java.util.Collection;
+
+import org.infinispan.commons.marshall.ProtoStreamTypeIds;
+import org.infinispan.marshall.protostream.impl.MarshallableArray;
+import org.infinispan.marshall.protostream.impl.MarshallableCollection;
+import org.infinispan.marshall.protostream.impl.MarshallableMap;
+import org.infinispan.marshall.protostream.impl.MarshallableObject;
+import org.infinispan.protostream.annotations.ProtoFactory;
+import org.infinispan.protostream.annotations.ProtoTypeId;
 import org.infinispan.remoting.transport.Address;
 
+@ProtoTypeId(ProtoStreamTypeIds.BIAS_REVOCATION_RESPONSE)
 public class BiasRevocationResponse extends SuccessfulResponse {
```

Comment:
No usage found in the IDE. Can be removed.

Reply:
I've created a PR for `main`: https://github.com/infinispan/infinispan/pull/13877